src/color-picker/color-picker.scss

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../menu/index';
+
+@mixin iui-color-picker {
+  @include iui-dropdown;
+  width: $iui-baseline * 27;
+  border-radius: $iui-border-radius;
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: $iui-sm;
+  padding-top: $iui-sm;
+
+  &.iui-advanced {
+    width: $iui-baseline * 24;
+  }
+}
+
+@mixin iui-color-option {
+  @include iui-focus;
+  height: $iui-l;
+  width: $iui-l;
+  margin-right: $iui-sm;
+  margin-bottom: $iui-sm;
```

Comment:
why only right/bottom?

Reply:
I think this is a strong case for flex `gap` (#178).

For now we could use `gap` inside `@supports` and fallback to right/bottom margins

Reply:
@bentleyvk [left a comment here](https://github.com/iTwin/iTwinUI/discussions/178#discussioncomment-1225992) and is unsure if this is a good idea or not.